src/app/organization/holidays/edit-holiday/edit-holiday.component.html

```diff
@@ -0,0 +1,79 @@
+<div class="container">
```

Comment:
![Screenshot from 2020-05-30 17-51-01](https://user-images.githubusercontent.com/54709463/83328084-2fe09000-a29e-11ea-87eb-57982da4239a.png)
@BLasan in your last PR you had deleted the hollidays[] array please restore it as a part of this PR.

Reply:
I'll check that

Reply:
Can you mention the tested holiday id ?

Reply:
https://github.com/openMF/web-app/commit/8c9bb25fd807ff1965b474851e3a61426064bfaf
restore array on line 72

Reply:
added that line. Shall I push that ? But it's already been merged